client/types.go

```diff
@@ -134,7 +134,6 @@ type RPCTransaction struct {
 }
 
 type RosettaTxReceipt struct {
-	Type           uint8  `json:"type,omitempty"`
```

Comment:
This cannot be removed IMO. This is added by @xiaying-peng for https://github.com/mdehoog/op-rosetta